README.md

```diff
@@ -91,13 +91,13 @@ https://decred.org/downloads/
   to download, build, and install without needing to clone the repo. Run:
 
   ```sh
-  $ go install decred.org/dcrwallet/v3@master
+  $ go install decred.org/dcrwallet/v4@master
   ```
 
   to build the latest master branch, or:
 
   ```sh
-  $ go install decred.org/dcrwallet/v3@latest
+  $ go install decred.org/dcrwallet/v4@latest
```

Comment:
we haven't released anything v4 yet so keep this as v3.

Reply:
Done